packages/manager/modules/dbaas-logs/src/logs/detail/aliases/home/logs-aliases-home.html

```diff
@@ -21,86 +24,78 @@ <h3 data-translate="logs_aliases_title"></h3>
         </oui-datagrid-topbar>
         <oui-datagrid-column
             title="::'logs_aliases_col_title' | translate"
-            property="info.name"
+            property="name"
             sortable="asc"
```

Comment:
Prefix it on all the properties
```suggestion
            data-sortable="asc"
```